logging/test/logger_test.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by ybuchnik on 2/6/20.
+//
+#include "gtest/gtest.h"
+#include "Logger.hpp"
+using namespace concordlogger;
+TEST(logger_mdc_test, mdc_test) {
+  auto logger = Log::getLogger("test");
+  MDC::put(logger, "cid", "123456");
+  MDC::put(logger, "1", "2");
+  LOG_INFO(logger, "It works :)");
```

Comment:
This test isn't really automated, as it requires a user to look at the output. Can you automate it to check that the context is output as you expect? You can do this any number of ways. If you log to a file you can grep it, etc... Or you can ensure your log output goes to a string you can search.

Reply:
I removed this test.
I think that for logging, we can see that it works in the logs, so we don't need such a simple test.